src/Elcodi/Plugin/ProductCsv/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "elcodi-plugins/product-csv",
+    "license": "MIT",
+    "type": "elcodi-plugin",
+    "description": "Elcodi plugin to add Product import/export from .csv files",
+    "authors": [
+        {
+            "name": "Berny Cantos",
+            "email": "be@rny.cc"
+        },
+        {
+            "name": "Elcodi Community",
+            "homepage": "https://github.com/elcodi-plugin/product-csv/contributors"
+        }
+    ],
+    "require": {
+        "goodby/csv": "~1.0",
```

Comment:
Right, my bad. I'm looking for classes used in files changed.
```shell
grep -hw ^use --include=*.php -- $(git diff master..HEAD --name-only) | sort
use DateTime;
use Elcodi\Bundle\CoreBundle\DependencyInjection\Abstracts\AbstractExtension;
use Elcodi\Component\Plugin\EventInterface;
use Elcodi\Component\Product\Entity\Interfaces\ProductInterface;
use Elcodi\Plugin\ProductCsv\DependencyInjection\ProductCsvExtension;
use Goodby\CSV\Export\Protocol\ExporterInterface;
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
use Symfony\Bundle\FrameworkBundle\Controller\Controller;
use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\HttpFoundation\ResponseHeaderBag;
use Symfony\Component\HttpFoundation\StreamedResponse;
use Symfony\Component\HttpKernel\Bundle\Bundle;
use Twig_Environment;
```
So we need to require, at least, `elcodi/product`, `symfony/framework-extra-bundle`, `symfony/framework-bundle`, `symfony/http-foundation`, `symfony/http-kernel` and `twig/twig`.

As it needs the `elcodi.repository.product` service, replace `elcodi/product` with `elcodi/product-bundle`.

And require `symfony/dependency-injection` as @mmoreram suggests, so the default adapter works flawlessly.